src/components/DetailedRouteRow/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Box, Typography, IconButton } from '@mui/material';
+import NoteIcon from '@mui/icons-material/Note';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import DeleteIcon from '@mui/icons-material/Delete';
+import Status from '../Status';
+import { COLORS } from '@/constants/colors';
+import { FONT } from '@/constants/font';
+
+interface DetailedRouteRowProps {
+  city: string;
+  time: string;
+  status: 'completed' | 'failed' | 'not_arrived' | 'at_risk' | 'upcoming';
```

Comment:
move to enum and to constants, I think we will reuse this enum in multiple places later

Reply:
Got it